project/webapp/static/js/account.js

```diff
@@ -382,7 +382,9 @@ cw.AccountView = BB.View.extend({
 	getLegislators: function(coordinates){
 		var _this = this;
 		$.ajax({
-			url: "http://openstates.org/api/v1/legislators/geo/?long=" + coordinates.lng + "&lat="+ coordinates.lng + "&fields=full_name", headers:{"X-APIKEY": this.OPENSTATES_API_KEY}, dataType: "jsonp",
+			url: "~/legislation/openstates/pyopenstates.py",
```

Comment:
There is no need for a `.py` suffix for this URL.

Make sure you are actively studying and following Django conventions, such as using [urlpatterns](https://docs.djangoproject.com/en/2.1/intro/tutorial03/) to map functions to routes.